lib/Elastica/Query/DistanceFeature.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Elastica\Query;
+
+class DistanceFeature extends AbstractQuery
```

Comment:
Would be nice to have a link here to the elasticsearch docs for it.

Reply:
done ;)